<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FifthDialog</class>
 <widget class="QDialog" name="FifthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/main/images/biankuang_5.png);
background-color: rgb(170, 170, 127);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/bia1nkuang_01.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>已完成所有题目</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>81</x>
     <y>50</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/bia1nkuang_01.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>等待对手完成</string>
   </property>
  </widget>
  <widget class="QLabel" name="flash">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>94</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/backgr1ound3.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tap1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/bia1nkuang_01.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="tap2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/main/images/bia1nkuang_01.png);
font: 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Agency FB&quot;;</string>
   </property>
   <property name="text">
    <string>返回主菜单</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
